src/components/Select/Select.module.scss

```diff
@@ -15,9 +15,7 @@
 
     .StyledReactSelect__multi-value {
       margin: 0;
-      padding: 3px;
-      margin: 0;
-      padding: 3px;
+      padding: 2px;
       margin-left: 0;
```

Comment:
Only change I made to the existing styling:
1) You had duplicate styles in here
2) I made the padding for the multi-value boxes a tiny bit smaller, so it could fit without looking too crowded.

Let me know if that wont work and I can try to see if I can move around without changing this padding